hw-model/src/model_emulated.rs

```diff
@@ -58,10 +60,13 @@ pub struct ModelEmulated {
     cpu_enabled: Rc<Cell<bool>>,
     trace_path: Option<PathBuf>,
 
+    #[cfg(feature = "coverage")]
```

Comment:
nit: Even though it would be unused, should we still accept the image tag as input just to make the interface consistent?